src/test/resources/GebConfig.groovy

```diff
@@ -17,7 +17,9 @@ environments {
 
 // Default if geb.env is not set to one of 'chrome', or 'chromeHeadless'
 driver = {
-    new ChromeDriver()
+    ChromeOptions options = new ChromeOptions()
```

Comment:
dubious about this change.  In the feature tests we normally allow chrome to fire up so that it is easy to see what is goin on and even set:
```quitCachedDriverOnShutdown = false``` so that it can be debugged afterwards

Reply:
the default when running from gradle will still be to do headless since it is defined that way in build.gradle